storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -88,7 +88,11 @@ func (c *PrometheusConverter) createAttributes(resource pcommon.Resource, attrib
 	c.scratchBuilder.Sort()
 	sortedLabels := c.scratchBuilder.Labels()
 
-	labelNamer := otlptranslator.LabelNamer{UTF8Allowed: settings.AllowUTF8}
+	labelNamer := otlptranslator.LabelNamer{
+		UTF8Allowed:                 settings.AllowUTF8,
+		UnderscoreLabelSanitization: settings.LabelNameUnderscoreSanitization,
+		PreserveMultipleUnderscores: settings.LabelNamePreserveMultipleUnderscores,
```

Comment:
```suggestion
		UnderscoreLabelSanitization: true,
		PreserveMultipleUnderscores: true,
```

I would just not add those options to the settings struct, I don't see us making it configurable in the future